src/js/Content/Features/Community/Inventory/FInventoryMarketHelper.js

```diff
@@ -14,13 +14,16 @@ export default class FInventoryMarketHelper extends Feature {
 
             /* eslint-disable no-undef, camelcase */
             window.SteamFacade.jq(document).on("click", ".inventory_item_link, .newitem", () => {
-                if (!g_ActiveInventory.selectedItem.description.market_hash_name) {
-                    g_ActiveInventory.selectedItem.description.market_hash_name = g_ActiveInventory.selectedItem.description.name;
-                }
-                let market_expired = false;
-                if (g_ActiveInventory.selectedItem.description) {
-                    market_expired = g_ActiveInventory.selectedItem.description.descriptions.reduce(
-                        (acc, el) => (acc || el.value === "This item can no longer be bought or sold on the Community Market."),
+
+                // https://github.com/SteamDatabase/SteamTracking/blob/b3abe9c82f9e9d260265591320cac6304e500e58/steamcommunity.com/public/javascript/economy_common.js#L161
+                const marketHashName = g_ActiveInventory.selectedItem.description.market_hash_name
+                    || g_ActiveInventory.selectedItem.description.market_name
+                    || g_ActiveInventory.selectedItem.description.name;
```

Comment:
Why not use the existing `GetMarketHashName` function?

Reply:
yeah why not😂